build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -266,5 +269,25 @@ private static void FixPermissions(string directory)
                 FS.FixModeFlags(directory);
             }
         }
+
+        private static string GetSharedFrameworkPublishPath()
+        {
+            string sharedFrameworkPublishPath = string.Empty;
+            
+            string preBuiltPortableBinaryPath=Environment.GetEnvironmentVariable("PORTABLE_BINARY_LOCATION")?.Trim();
+
+            // set to default path if PORTABLE_BINARY_LOCATION environment variable doesn't exist
+            if(preBuiltPortableBinaryPath == null)
+            {
+                sharedFrameworkPublishPath = Dirs.SharedFrameworkPublish;
+            }
+            else 
+            {
+                Console.WriteLine($"Installers will package binaries from path set by PORTABLE_BINARY_LOCATION environment variable :{preBuiltPortableBinaryPath}");
+                sharedFrameworkPublishPath = preBuiltPortableBinaryPath;
```

Comment:
Is it okay to use Console.WriteLine() here or expected to call BuildContext.warn() ? I really want to add the log here.